docs/Extra Guides/qv.md

```diff
@@ -0,0 +1,155 @@
+member proportional to the square root of the number of tokens they hold. 
+
+This has the effect of increasing the voting power of minority voters compared to large token holders,
+increasing alignment and encouraging voter participation.
+
+## What is Sybil Resistance?
+
+One challenge to Quadratic Voting is the potential for a single voter to create multiple accounts, split their
+tokens between them, and vote multiple times.
+
+This is known as a **Sybil Attack**.
+
+To mitigate this in Realms, the [Civic Pass Plugin](civic.md) is also used, to ensure that each voter is a unique individual.
+
+This means that Quadratic Voting DAOs in Realms require voters to have a Civic Pass. For more details, see [civic.com](https://civic.com).
+
+## How to use Quadratic Voting in Realms?
+
+### Voting in a Quadratic Voting DAO
+
+As a voter, once you have obtained your Civic Pass (see above), 
+
+### Creating a QV DAO from Scratch
+
+##### Step 1 - Select Community DAO
+
+Choose the **Community Token DAO** option from the **Create DAO** page.
+
+![community-dao.png](./QV/community-dao.png)
+
+##### Step 2 - Select Quadratic
+
+In the **Community Token** section, select **Advanced Options**,
+and then select **Quadratic**.
+
+![select-quadratic.png](./QV/select-quadratic.png)
+
+Note: It is possible here to configure the Quadratic Voting plugin. We recommend keeping the default settings.
+To learn about the configuration options, see below.
+
+##### Step 3 - Create DAO
+
+Proceed through the other steps of the DAO and create it.
+
+![create-dao.png](./QV/create-dao.png)
+
+### Adapting an existing DAO to use QV
+
+:::danger
+The **spl-governance** program version must be equal or higher than **v2.2.6** for the **QV** plugin to work correctly.
+
+If the plugin is enabled for **older versions** it can result in **irreversible deadlock** of the **DAO**.
+:::
+
+
+#### Step 1 - Add Civic Pass Plugin
+
+Select the **Params** button and choose **Change config**.
+
+![params.png](./QV/params.png)
+
+Select **Civic** as the governance structure
+
+![edit-realm-civic.png](./QV/edit-realm-civic.png)
+
+Leave the pass type as the default.
+
+Click **Continue**, **Create Proposal** and vote on the resultant proposal.
+
+Once the proposal has passed, execute it.
+
+#### Step 2 - Add QV Plugin
+
+Once again, select the **Params** button and choose **Change config**.
+
+This time, select **Quadratic** as the governance structure, and enable the
+**Chain this plugin with the Civic Plugin?** option.
+
+![chain-qv-plugin.png](./QV/chain-qv-plugin.png)
+
+Leave the coefficients as the default.
+
+Click **Continue**, **Create Proposal** and vote on the resultant proposal.
+
+Once the proposal has passed, execute it.
+
+You have now converted the DAO to use Quadratic Voting!
+
+## Advanced Configuration
+
+### Coefficients
+
+The QV plugin uses the following formula for calculating the voting power of a member:
+
+$$
+a\sqrt{x}+bx+c
+$$
```

Comment:
The mathematical notation is not correctly displayed 

<img width="232" alt="image" src="https://github.com/solana-labs/governance-docs/assets/5817016/edc4c92e-823f-49d6-9f19-599a0b7f3b8e">